src/components/AccountSelector/AccountSelectButton/AccountSelector.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import { FiUser } from "react-icons/fi";
+import { MassaToken } from "../../../assets/svg-components/MassaToken";
+import { AccountSelectorButton } from "./AccountSelectorButton";
+
+export default {
+  title: "Components/Account Selector",
+  component: AccountSelectorButton,
+};
+
+const theme = "theme-light";
+
+const args = {
+  profileimage: <FiUser className="text-neutral h-6 w-6" />,
```

Comment:
Is it left up to the developer to change the size of the icon outside the component?

If this is not the case, the classnames should be moved into the component.

Suggestion: you can remove class `text-neutral` because it's just a picture and not a text.
```typescript
profileimage: <FiUser className="h-6 w-6" />,
```